setup/features/server-group-launch-settings/aws-ec2/launch-templates.md

```diff
@@ -0,0 +1,322 @@
+---
+layout: single
+title:  "AWS Launch Templates"
+sidebar:
+nav: setup
+---
+
+{% include toc %}
+
+## Enable launch template feature set in Spinnaker
+Follow guidelines [here](/features/server-group-launch-settings/aws-ec2/launch-templates-setup.md)
+
+Note: The features supported in Deck come with helpful tool tips that aid in learning about them quickly.
+Consider trying them out in Deck before using them in API, especially if you are new to them.
+
+## Launch Template Feature Configuration
+Once launch templates are enabled in Clouddriver, a new set of features are unlocked.
+Some of them are tied to the Launch Template directly like `IMDSV2` and 
+some others require a launch template to be used in order to support features like `SpotAllocationStrategy` ([MixedInstancesPolicy](https://docs.aws.amazon.com/autoscaling/ec2/APIReference/API_MixedInstancesPolicy.html) features).
+
+
+Review the sections below for details about the features and sample use cases along with Spinnaker API requests. 
+
+### Launch Template Only Parameters
+Review the sections below to determine which features you want to enable in the UI.
```

Comment:
We usually say "console" rather than "UI". I'm not sure what "UI" refers to here?